sources/install/package_ad.sh

```diff
@@ -1366,6 +1366,14 @@ function install_sccmwtf() {
     add-to-list "sccmwtf,https://github.com/xpn/sccmwtf,This code is designed for exploring SCCM in a lab."
 }
 
+function install_smbclientng() {
```

Comment:
@p0dalirius, can you add : # CODE-CHECK-WHITELIST=add-aliases

Reply:
Done